src/content_mapblock.cpp

```diff
@@ -172,6 +172,13 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+namespace
```

Comment:
Please make the function static instead of using unnamed namespaces. It has the same outcome, but is more used in the code, and less "C++y".